safe-global/web-core
# fix: fallback to "Contract interaction" for module

## What it solves

Resolves #1297

## How this PR fixes it

"Contract interaction" is used as a fallback for unknown module interactions.

## How to test it

Open `eth:0x0DA0C3e52C977Ed3cBc641fF02DD271c3ED55aFe/transactions/tx?id=module_0x0DA0C3e52C977Ed3cBc641fF02DD271c3ED55aFe_0xe14278097fc7d0f462d2913d36185505ee09f8476ac899620c409eb0afaa812a_0xa65aaf534f1a4620` on production and observe the type be "Contract interaction".

## Screenshots

![image](https://user-images.githubusercontent.com/20442784/205113670-43b1237e-78c2-41be-9b4e-2d004a1a69e2.png)
